trunner/runners/common.py

```diff
@@ -308,21 +310,58 @@ def go(self):
 class Runner:
     """Common interface for test runners"""
 
+    @abstractmethod
+    def status(self, status):
+        """Method for sygnalising a current runner status: busy/failed/succeeded"""
+        pass
+
+    @abstractmethod
     def flash(self):
         """Method used for flashing a device with the image containing tests."""
         pass
 
+    @abstractmethod
     def run(self, test):
         """Method used for running a single test case which is represented by TestCase class."""
         pass
 
 
 class DeviceRunner(Runner):
-    """This class provides interface to run test case using serial port"""
+    """This class provides interface to run tests on hardware targets using serial port
+       Hardware targets are run on Rpi
+       GPIO 13 must be connected to a red LED
+       GPIO 18 must be connected to a green LED
+       GPIO 12 must be connected to a blue LED"""
 
     def __init__(self, port):
         self.port = port
         self.serial = None
+        self.ledr_gpio = GPIO(13)
+        self.ledg_gpio = GPIO(18)
+        self.ledb_gpio = GPIO(12)
+        self.ledb_gpio.high()
+
+    def status(self, status):
+        if status == 'BUSY':
+            self.set_led('blue')
+        if status == 'SUCCESS':
+            self.set_led('green')
+        if status == 'FAIL':
+            self.set_led('red')
+
+    def set_led(self, color):
+        colors = ['red', 'green', 'blue', 'none']
+
+        if color in colors:
+            self.ledr_gpio.low()
+            self.ledg_gpio.low()
+            self.ledb_gpio.low()
+            if color == 'red':
+                self.ledr_gpio.high()
+            if color == 'green':
+                self.ledg_gpio.high()
+            if color == 'blue':
+                self.ledb_gpio.high()
```

Comment:
More `python` style:
```suggestion
        self.leds = {'red': GPIO(13), 'green': GPIO(18), 'blue': GPIO(12)}
        self.status_color = {'BUSY': 'blue', 'SUCCESS': 'green', 'FAIL': 'red'}

        self.leds['blue'].high()

    def status(self, status):
        if status in self.status_color:
            self.set_led(self.status_color[status])

    def set_led(self, color):
        if color in self.leds:
            for v in self.leds.values():
                v.low()
            self.leds[color].high()
```

Reply:
Thanks, changes applied